app/scripts/controllers/console.js

```diff
@@ -45,7 +45,78 @@ const crate_console = angular.module('console', ['sql', 'datatypechecks', 'stats
 
     return service;
   })
-  .directive('console', function(SQLQuery, ColumnTypeCheck, ConsoleFormatting, ClusterState){
+  .service('ExportUtils', function () {
+    var utils = {};
+
+    utils.arrayFrom = function (json) {
```

Comment:
It would be great if you add tests to the functions of this service. We keep the test files in /tests folder